pkg/azure/conversion_worker.py

```diff
@@ -84,9 +95,25 @@ async def conversion_worker(osdisk_raw,project,host):
             vhd_path = "{}/projects/{}/{}/osdisks/{}".format(cur_path, project, host, osdisk_vhd)
             print("Start converting")
             os.popen('echo "start converting">> ./logs/ansible/migration_log.txt')
-            command2 = f"qemu-img convert -f raw -o subformat=fixed,force_size -O vpc {path} {vhd_path}"
-            process2 = await asyncio.create_subprocess_shell(command2, stdin = PIPE, stdout = PIPE, stderr = STDOUT)
-            await process2.wait()
+            MB = 1024 * 1024
+            size = await get_size(path)
+            rounded_size = ((size // MB) + 1) * MB
+            if size % MB == 0:
+                logger("Size is already rounded","info")
+                command = f"qemu-img convert -f raw -o subformat=fixed,force_size -O vpc {path} {vhd_path}"
+                process = await asyncio.create_subprocess_shell(command, stdin = PIPE, stdout = PIPE, stderr = STDOUT)
+                await process.wait()
```

Comment:
Can this be rewritten as `await asyncio.create_subprocess_shell(command, stdin = PIPE, stdout = PIPE, stderr = STDOUT).wait()` ? would be much cleaner if possible.